src/AcquiaDrupalEnvironmentDetector.php

```diff
@@ -193,6 +193,28 @@ public static function getAcsfDbName() {
     return isset($GLOBALS['gardens_site_settings']) && self::isAcsfEnv() ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
   }
 
+  /**
+   * Get a standardized site / db name.
+   *
+   * On ACE or simple multisite installs, this is the site directory under
+   * 'docroot/sites'.
```

Comment:
Does this mean it will return `default` on most single site installs?

Reply:
Yes. This is how I'm leveraging it to find the automatically generated settings files within ACE:

`/**
 * Acquia Cloud automatically generates a settings file that contains the
 * database credentials for a given Drupal application, along with some other
 * required plumbing. The code below uses a plugin to load the correct file
 * based on the current product, site, and environment.
 */
$site_name = AcquiaDrupalEnvironmentDetector::getSiteName($site_path);
if (AcquiaDrupalEnvironmentDetector::isAhEnv()) {
  $ah_group = AcquiaDrupalEnvironmentDetector::getAhGroup();
  if (!AcquiaDrupalEnvironmentDetector::isAcsfEnv()) {
    if ($site_name == 'default') {
      require "/var/www/site-php/$ah_group/$ah_group-settings.inc";
    } else {
      // Acquia Cloud does not support periods in db names.
      $safe_site_name = str_replace('.', '_', $site_name);
      require "/var/www/site-php/$ah_group/$safe_site_name-settings.inc";
    }
  }
}`